Task_One/Firest_task.rb

```diff
@@ -0,0 +1,20 @@
+OPTIMAL = 110
+
+class IdealWeight
+  
+  def ideal_wright(name,height)
+    if height.to_i - OPTIMAL <= 0
+      print "#{name} Your weight is already optimal"
+    else
+      optimal_weight = height.to_i - OPTIMAL 
+      print "#{name} Your optimal weight is #{optimal_weight}"
+    end
+  end
+end
+
+idealweight = IdealWeight.new
+
+name = gets.chomp
+height = gets.chomp
+
+puts idealweight.ideal_wright(name, height)
```

Comment:
all text files should end with a newline
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline#:~:text=If%20content%20is%20added%20to,you%20actually%20wanted%20to%20see.

Reply:
I ended it with new line but git deleted it
and now I added multiple new lines in my cod